freemocap/core_processes/detecting_things_in_2d_images/mediapipe_stuff/mediapipe_skeleton_detector.py

```diff
@@ -74,7 +74,7 @@ def process_single_video(
         logger.info(f"Running `mediapipe` skeleton detection on video: {str(synchronized_video_file_path)}")
 
         holistic_tracker = mp_holistic.Holistic(
-            model_complexity=mediapipe_parameters_model.model_complexity,
+            mediapipe_model_complexity=mediapipe_parameters_model.mediapipe_model_complexity,
```

Comment:
@CypressJack All the other changes look good, but "model_complexity" is a keyword argument for mediapipe holistic, so we can't change the left side of the equality here. Our tests caught it - to run the tests locally you can run "pytest freemocap/tests" in the freemocap folder and it will do a headless run through on sample data

Reply:
@philipqueen My apologies for missing that, the local tests pass now and the correction has been pushed renaming the param to `model_complexity`